config/common/files/CONTRIBUTING.md.mustache

```diff
@@ -33,7 +33,7 @@ That repo includes an issue template that will walk through all the places to ch
 
 ### Submitting Pull Requests
 
-Considering that the SDKs are autogenerated, please make sure to submit your Pull Requests to the [sdk-generator](https://{{gitHost}}/{{gitUserId}}/sdk-generator). We will not accept PRs to this repository because they will be overwritten on the next sdk generation.
+While we accept Pull Requests on this repository, the SDKs are autogenerated so please consider submitting your Pull Requests to the [sdk-generator](https://{{gitHost}}/{{gitUserId}}/sdk-generator) which assists the OpenFGA team in deploying fixes and updates to our other SDKs as well. 
```

Comment:
```suggestion
While we accept Pull Requests on this repository, the SDKs are autogenerated so please consider additionally submitting your Pull Requests to the [sdk-generator](https://{{gitHost}}/{{gitUserId}}/sdk-generator) and linking the two PRs together and to the corresponding issue. This will greatly assist the OpenFGA team in being able to give timely reviews as well as deploying fixes and updates to our other SDKs as well. 
```